### PR TITLE
Release 4.10.0 release

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by damoore044** [comment]
Had @chris1984 open this PR since I used his credentials to create the branch, thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,89 @@
...
+
+## Bug Fixes
+
+### Web UI
+ * about page broken after katello-agent removal ([#36722](https://projects.theforeman.org/issues/36722), [5d2f049c](https://github.com/Katello/katello.git/commit/5d2f049c72b900f45de7e062edc162207b6743ba))
+ * Legacy Content Host UI - Remove katello-agent code ([#36649](https://projects.theforeman.org/issues/36649), [5dc6942e](https://github.com/Katello/katello.git/commit/5dc6942e738d55c0a4a672cab881245aa482f3ed))
+ * Content tab subtabs disappear from host details page when you click them ([#36613](https://projects.theforeman.org/issues/36613), [d12f0b13](https://github.com/Katello/katello.git/commit/d12f0b1303608401912e2e28e3f1f2ce1f53ae5e))
+
+### Hosts
+ * Discovery Provisioning fails as the sync media is getting diminished from the host page even though its synced. ([#36721](https://projects.theforeman.org/issues/36721), [58dd1b73](https://github.com/Katello/katello.git/commit/58dd1b73e79ce27895107adbfaa5d7122a95204b))
```
Comment (ianballou) [inline CHANGELOG.md:36]: I think some of these changelog entries are duplicates: https://github.com/Katello/katello/pull/10759/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR119

**Review by ianballou — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (chris1984) [reply CHANGELOG.md:36]: Will pull down the changelog and fix it up and repush, not sure why it did that. Good catch

Comment (damoore044) [reply CHANGELOG.md:36]: nice catch thanks! @ianballou

**Comment by ianballou** [comment]
I think up until GA, for the RCs, the changelog needs to be completely regenerated. But then once we do z-releases then the changelog can get appended to.

**Review by ianballou — APPROVED** [review]
Looking good to me!

**Pull request merged**